core/storage/user/gae_models.py

```diff
@@ -856,13 +856,22 @@ def export_data(user_id):
         if user_model is None:
             return {}
 
+        creator_usernames = []
+        creator_user_models = UserSettingsModel.get_multi(
+            user_model.creator_ids)
+
+        for creator in creator_user_models:
+            creator_usernames.append(creator.username)
+
         user_data = {
             'activity_ids': user_model.activity_ids,
             'collection_ids': user_model.collection_ids,
             'general_feedback_thread_ids': (
                 user_model.general_feedback_thread_ids),
-            'creator_ids': user_model.creator_ids,
-            'last_checked': user_model.last_checked
+            'creator_usernames': creator_usernames,
+            'last_checked':
+                None if user_model.last_checked is None else
+                utils.get_time_in_millisecs(user_model.last_checked)
```

Comment:
```suggestion
            'last_checked':
                utils.get_time_in_millisecs(user_model.last_checked) 
                if user_model.last_checked else None
```

Reply:
Should I be using truthy values? In the past I think I've received comments not to use 'if user_model.last_checked' and instead use 'if not user_model.last_checked is None'

Reply:
I think what @varun-tandon has here is fine. @vojtechjelinek are you ok with it?

(In this case I think both approaches would probably work. The "if not None" approach is more "correct", but it does get a bit wordy in Vojta's example to say "if x is not None else None".)

Reply:
Yep, fine with it.